contracts/src/LightClient.sol

```diff
@@ -121,6 +133,16 @@ contract LightClient is Initializable, OwnableUpgradeable, UUPSUpgradeable {
         BN254.ScalarField blockCommRoot;
     }
 
+    /// @notice Simplified HotShot commitment struct
+    /// @param l1BlockHeight the block height of l1 when this state update was stored
+    /// @param hotshotBlockHeight The block height of the latest finalized HotShot block
```

Comment:
The params look outdated.

Reply:
that's correct, i've updated it here https://github.com/EspressoSystems/espresso-sequencer/commit/f36a7c41893c792fe780077fd4e23aa57aa988e4